package.json

```diff
@@ -228,14 +228,16 @@
     "check": "mongodb-js-precommit './src/**/*{.js,.jsx}' './test/**/*.js'",
     "clean": "hadron-build clean",
     "compile-ui": "hadron-build ui",
-    "release": "hadron-build release",
+    "release-evergreen": "hadron-build release",
+    "release": "node release/index.js",
     "test-unit": "npm test -- --unit",
     "test-enzyme": "npm test -- --enzyme",
     "test-packages": "npm test -- --packages",
     "test-main": "npm test -- --main",
     "test-renderer": "npm test -- --renderer",
     "test-functional": "npm test -- --functional",
     "test-release": "npm test -- --release",
+    "test-release-tasks": "mocha --timeout 30000 release/*.spec.js",
```

Comment:
Do these tests run automatically ever?  Do we want them to?

Reply:
I have no idea. Hopefully once we switch to a monorepo that will go in its own folder and tests will be run as they are for any other package. I'm not sure i can follow the whole `hadron-build` mechanics yet.